examples/atari/dqn/README.md

```diff
@@ -22,70 +22,95 @@ python train_dqn.py [options]
 To view the full list of options, either view the code or run the example with the `--help` option.
 
 ## Results
-These results reflect ChainerRL  `v0.5.0`.
```

Comment:
What do you mean by `v0.5.0`/`v0.6.0`? I think it should be `v0.6.0` since `v0.5.0` does not have time-based evaluation.

Reply:
You're correct, these results were obtained using ChainerRL `v0.5.0` as well as ChainerRL `v0.6.0`, but it seems that all the features I was using were integrated in ChainerRL `v0.6.0`. I will change it.